### PR TITLE
BEP016: Initial draft of common filename suffix "model"

### PR DESCRIPTION
**Pull request by Lestropie**
Draft PR in order to discuss issues described in greater detail in #32.

### CODE REVIEW

**Comment by Lestropie** [comment]
Any opinions on merging this prior to greater broadcast?
It may not yet be resolved in #50, but I'm progressively leaning more toward this; and [additionally from @sappelhoff](https://github.com/bids-standard/bids-bep016/issues/50#issuecomment-1202173036):

> Re: Decision 2 --> I think one of the principles in BIDS so far was to use as few suffixes as possible, as many as needed ... so that makes Option 1 appear more favorable for me.

File: src/05-derivatives/05-diffusion-derivatives.md [hunk 1]
```diff
@@ -435,11 +427,15 @@ another.
...
-                sub-01_dti.json
+                sub-01_desc-dti_parameter-all_model.nii.gz
+                sub-01_desc-dti_parameter-bzero_model.nii.gz
+                sub-01_desc-dti_parameter-fa_mdp.nii.gz
+                sub-01_desc-dti_model.json
     ```
 
-    Dimensions of NIfTI image "`sub-01_dti.nii.gz`": *I*x*J*x*K*x6 ([parameter vectors](#data-param))
+    Dimensions of NIfTI image "`sub-01_desc-dti_parameter-all_model.nii.gz`": *I*x*J*x*K*x6 ([parameter vectors](#data-param))
+    Dimensions of NIfTI image "`sub-01_desc-dti_parameter-bzero_model.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
```
Comment (arokem) [inline src/05-derivatives/05-diffusion-derivatives.md:437]: On the one hand, the first file among these two has an "all" in its file name suggesting that all relevant parameters are found in this file. On the other hand, we immediately have another `_model` file right next to it. That's a bit confusing, isn't it? So either the b0 goes into the `_all_model` file, or we need to find another way to find all the relevant model parameters distributed across multiple files.

**Review by arokem — COMMENTED** [review]
After merging #51, I think that you will need to adjust "parameter" to "param" wherever that appears. I think that it would be good to continue the discussion about the "all" designation, which I highlight in my comment here. Will open a separate discussion, referencing the relevant PRs (this one and #52).

File: src/05-derivatives/05-diffusion-derivatives.md (hunk 1 above)
Comment (arokem) [reply src/05-derivatives/05-diffusion-derivatives.md:437]: But otherwise, I think this is good to go.

Comment (Lestropie) [reply src/05-derivatives/05-diffusion-derivatives.md:437]: That's an error on my part. "`_param-all`" should only appear if it's the only image containing model parameters. If the tensor fitting also yields a *b*=0 estimate, it needs to be something like "`_param-tensor`" and "`_param-bzero`".

Comment (Lestropie) [reply src/05-derivatives/05-diffusion-derivatives.md]: Indeed exactly this was already described in the prior table. Should be resolved in https://github.com/bids-standard/bids-bep016/pull/46/commits/d3bbb8686a3b4c6298bcee29f4df51e199dfc720.

**Comment by francopestilli** [comment]
We agree to merge this and now we can start with the next

**Pull request merged**